extensions/jaxp/deployment/src/main/java/io/quarkus/jaxp/deployment/JaxpProcessor.java

```diff
@@ -30,15 +31,16 @@ void reflectiveClasses(BuildProducer<ReflectiveClassBuildItem> reflectiveClass)
 
     @BuildStep
     void resourceBundles(BuildProducer<NativeImageResourceBundleBuildItem> resourceBundle) {
+        Consumer<String> resourceBundleItemProducer = bundleName -> resourceBundle
+                .produce(new NativeImageResourceBundleBuildItem(bundleName, "java.xml"));
         Stream.of(
```

Comment:
These changes are fixing: https://github.com/graalvm/mandrel/actions/runs/3156296401/jobs/5136091100#step:11:343, but we also need https://github.com/oracle/graal/pull/5118 correct?

Reply:
Correct